### PR TITLE
Fix a few grammar issues and typos

### PR DESCRIPTION
**Pull request by mehabhalodiya**
These frequent grammar errors might be making you look bad. I have fixed these common mistakes and avoid damaging your credibility.

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
@kbhawkey Can you please review?
Let me know if anything needs to be changed.

File: content/en/docs/contribute/advanced.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ refer to
 The SIG Docs representative for a given release coordinates the following tasks:
 
 - Monitor the feature-tracking spreadsheet for new or changed features with an
-  impact on documentation. If documentation for a given feature won't be ready
+  impact on documentation. If the documentation for a given feature won't be ready
```
Comment (sftim) [inline content/en/docs/contribute/advanced.md:57]: The original wording is also idiomatic. It's OK to add a definite article here.

File: content/en/docs/contribute/advanced.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ in the Kubernetes organization. The contributor's membership needs to be
 backed by two sponsors who are already reviewers.
 
 New docs contributors can request sponsors by asking in the #sig-docs channel
-on the [Kubernetes Slack instance](https://kubernetes.slack.com) or on the
```
Comment (sftim) [inline content/en/docs/contribute/advanced.md:104]: I prefer the original wording here. “Ask on the mailing list” is using “on”, whereas “ask in the #sig-docs” channel is using “in”.

File: content/en/docs/contribute/advanced.md [hunk 3]
```diff
@@ -122,7 +122,7 @@ Approvers must meet the following requirements to be a co-chair:
 - Understand SIG Docs workflows and tooling: git, Hugo, localization, blog subproject
 - Understand how other Kubernetes SIGs and repositories affect the SIG Docs
   workflow, including:
-  [teams in k/org](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml),
+  [teams in k/org](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml), the
```
Comment (sftim) [inline content/en/docs/contribute/advanced.md:125]: If changing “process” we should also change “plugins”. I'd prefer to change neither though.

File: content/en/docs/contribute/advanced.md [hunk 4]
```diff
@@ -135,11 +135,11 @@ The role of co-chair is one of service: co-chairs build contributor capacity, ha
 Responsibilities include:
 
 - Keep SIG Docs focused on maximizing developer happiness through excellent documentation
-- Exemplify the [community code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and hold SIG members accountable to it
+- Exemplify the [community code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and hold SIG members accountable for it
```
Comment (sftim) [inline content/en/docs/contribute/advanced.md:138]: (I assume that) the elided phrase is “comply with”, so “accountable to &hellip; it” is correct.

File: content/en/docs/contribute/advanced.md [hunk 5]
```diff
@@ -135,11 +135,11 @@ The role of co-chair is one of service: co-chairs build contributor capacity, ha
...
 - Learn and set best practices for the SIG by updating contribution guidelines
 - Schedule and run SIG meetings: weekly status updates, quarterly retro/planning sessions, and others as needed
 - Schedule and run doc sprints at KubeCon events and other conferences
-- Recruit for and advocate on behalf of SIG Docs with the {{< glossary_tooltip text="CNCF" term_id="cncf" >}} and its platinum partners, including Google, Oracle, Azure, IBM, and Huawei
+- Recruit for and advocate on behalf of SIG Docs with the {{< glossary_tooltip text= "CNCF" term_id= "cncf" >}} and its platinum partners, including Google, Oracle, Azure, IBM, and Huawei
```
Comment (sftim) [inline content/en/docs/contribute/advanced.md:142]: We usually write shortcodes with no space after `=`.

**Review by sftim — CHANGES_REQUESTED** [review]
I'm afraid I disagree with more of these changes than I agree with. Are they machine-generated?

File: content/en/docs/contribute/advanced.md [hunk 6]
```diff
@@ -26,8 +26,8 @@ may have ideas for improving the [Style Guide](/docs/contribute/style/style-guid
 the documentation, the website style, the processes for reviewing and merging
 pull requests, or other aspects of the documentation. For maximum transparency,
 these types of proposals need to be discussed in a SIG Docs meeting or on the
-[kubernetes-sig-docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
-In addition, it can really help to have some context about the way things
+[Kubernetes-sig-docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
```
Comment (reylejano) [inline content/en/docs/contribute/advanced.md:29]: The original is the correct name for the mailing list
```suggestion
[kubernetes-sig-docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
```

**Comment by reylejano** [comment]
Hi @mehabhalodiya , since comments have not been addressed and this PR has had no action for the past 2 months, I'm closing this PR.

Feel free to re-open this PR when you have addressed the comments.

/close

**Comment by mehabhalodiya** [comment]
Hi @reylejano 
I will make changes soon as till now I haven't seen the reviewed comments.
Sorry for inconvenience! Requesting you to open this!

**Comment by jimangel** [comment]
Thanks for continuing work on this!
/label tide/merge-method-squash
/lgtm
/approve

**Pull request merged**